user_docs/en/changes.t2t

```diff
@@ -23,6 +23,7 @@ What's New in NVDA
 - Added commands for toggling multiple modifiers simultaneously with a Braille display (#13152)
 - The Speech Dictionary dialog now features a "Remove all" button to help clear a whole dictionary. (#11802)
 - Added support for Windows 11 Calculator. (#13212)
+- Added new commands to jump to first/last row/column in tables.

```

Comment:
Please remove the change on this file. To avoid dealing with many merge conflicts on the change log, NVAccess prefers to update this file themselves. Once your PR is accepted, they will look at the change log section in the initial description of the PR and they will copy/paste its content to this file.